src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,43 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+};
+
+export const MovieCard: React.FC<Props> = ({ ...movie }) => (
```

Comment:
why do you use `rest` here, it leads to situation where you need to write `movie.` everywhere